src/confcom/azext_confcom/security_policy.py

```diff
@@ -909,116 +915,96 @@ def load_policy_from_str(
-                    + f'["{config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS}"]'
-                    + "can only be a string value."
-                )
-
-            iss = case_insensitive_dict_get(
-                fragment, config.ACI_FIELD_CONTAINERS_REGO_FRAGMENTS_ISS
-            ) or case_insensitive_dict_get(fragment, config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS_ISSUER)
-            if not isinstance(iss, str):
-                eprint(
-                    f'Field ["{config.ACI_FIELD_CONTAINERS}"]'
-                    + f'["{config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS}"]'
-                    + f'["{config.ACI_FIELD_CONTAINERS_REGO_FRAGMENTS_ISS}"]'
-                    + "can only be a string value."
-                )
-
-            minimum_svn = case_insensitive_dict_get(
-                fragment, config.ACI_FIELD_CONTAINERS_REGO_FRAGMENTS_MINIMUM_SVN
-            ) or case_insensitive_dict_get(fragment, config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS_MINIMUM_SVN)
-            if not isinstance(minimum_svn, str):
-                eprint(
-                    f'Field ["{config.ACI_FIELD_CONTAINERS}"]'
-                    + f'["{config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS}"]'
-                    + f'["{config.ACI_FIELD_CONTAINERS_REGO_FRAGMENTS_MINIMUM_SVN}"]'
-                    + "can only be a string value."
-                )
+    scenario = case_insensitive_dict_get(
+        policy_input_json, config.ACI_FIELD_SCENARIO
+    ) or ""
 
-            includes = case_insensitive_dict_get(
-                fragment, config.ACI_FIELD_CONTAINERS_REGO_FRAGMENTS_INCLUDES
-            )
-            if not isinstance(includes, list):
-                eprint(
-                    f'Field ["{config.ACI_FIELD_CONTAINERS}"]'
-                    + f'["{config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS}"]'
-                    + f'["{config.ACI_FIELD_CONTAINERS_REGO_FRAGMENTS_INCLUDES}"]'
-                    + "can only be a list."
-                )
+    # 3) Process rego_fragments
+    if rego_fragments:
+        process_fragment_imports(rego_fragments)
 
-    if not containers and not rego_fragments:
+    if not input_containers and not rego_fragments:
         eprint(
             f'Field ["{config.ACI_FIELD_CONTAINERS}"]' +
             f' and field ["{config.ACI_FIELD_CONTAINERS_REGO_FRAGMENTS}"] can not both be empty.'
         )
 
-    for container in containers:
-        image_properties = case_insensitive_dict_get(container, config.ACI_FIELD_TEMPLATE_PROPERTIES)
-        image_name = case_insensitive_dict_get(
-            container, config.ACI_FIELD_CONTAINERS_CONTAINERIMAGE
-        ) or case_insensitive_dict_get(image_properties, config.ACI_FIELD_TEMPLATE_IMAGE)
-
+    for container in input_containers:
         container_name = case_insensitive_dict_get(
             container, config.ACI_FIELD_CONTAINERS_NAME
-        ) or image_name
```

Comment:
Is image_name no longer a backup for container_name?

Reply:
Adding this back